tests/versioning.go

```diff
@@ -1677,16 +1677,16 @@ func (s *VersioningIntegSuite) waitForPropagation(ctx context.Context, tq, newBu
 	nsId := s.getNamespaceID(s.namespace)
 	s.Eventually(func() bool {
 		for pt := range remaining {
-			partName, err := tqname.FromBaseName(tq)
+			taskQueue, err := tqid.FromBaseName("", taskQueue)
 			s.NoError(err)
-			partName = partName.WithPartition(pt.part)
+			partition := taskQueue.NormalPartition(0, pt.part)
```

Comment:
shouldn't you pass pt.tp in here? if the partition now holds the type, it should hold the correct type